packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -878,10 +878,7 @@ const processAppExtensions = (
             const appExtensionDestDir = p.join(appExtensionsDir, appExtensionName.replace('/', '_'))
             sh.mkdir('-p', appExtensionDestDir)
 
-            // Copy hidden files
-            sh.cp('-rf', p.join(appExtensionTmpPath, '.*'), appExtensionDestDir)
-            // Copy regular files
-            sh.cp('-rf', p.join(appExtensionTmpPath, '*'), appExtensionDestDir)
+            copyAllFiles(appExtensionTmpPath, appExtensionDestDir)
```

Comment:
The changes in this file are new.. they're not from the `develop` branch. They fix the failing project generation on windows machine.

Otherwise, you would see an error like this:

![Arc 2025-04-30 at 22 39 09](https://github.com/user-attachments/assets/d904feb9-62e9-4f5f-8d52-dadf27738086)

Reply:
Should we add those changes back to the develop branch afterward?

Reply:
@vcua-mobify Good question. Fortunately we don't need to bring those changes back to develop. They're only needed for the extensibility branch as the issue is newly introduced there.